app/build.gradle

```diff
@@ -214,6 +214,10 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation "androidx.room:room-runtime:${androidxRoomVersion}"
     implementation "androidx.room:room-rxjava3:${androidxRoomVersion}"
+
+    // Apple Silicon Mac workaround (https://issuetracker.google.com/issues/174695268#comment9)
+    kapt "org.xerial:sqlite-jdbc:3.34.0"
+
```

Comment:
As in the others PRs, please remove this and only keep it locally
```suggestion
```

Reply:
Done in https://github.com/TeamNewPipe/NewPipe/pull/6919/commits/7e27e73532e3253a18a323421accccd4c662bd17